WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupGoogleFragment.java

```diff
@@ -151,6 +151,7 @@ public void onAuthenticationChanged(AccountStore.OnAuthenticationChanged event)
                      "SignupGoogleFragment.onAuthenticationChanged: " + event.error.type + " - " + event.error.message);
             // Continue with signup since account was created.
         } else if (event.createdAccount) {
+            AnalyticsTracker.track(AnalyticsTracker.Stat.CREATED_ACCOUNT);
```

Comment:
Hmm, it might be my impression but, isn't the track going to be triggered anyway when the epilogue shows up? Won't the `mGoogleListener.onGoogleSignupFinished()` below cause the epilogue to show up?

Reply:
With the changes in e1a88d6, we need to keep this trigger to ensure this analytic is tracked for Google signup.